wandb/sdk/wandb_run.py

```diff
@@ -1854,77 +1888,118 @@ def save(
             )
             return True
 
-        return self._save(glob_str, base_path, policy)
+        if isinstance(glob_str, bytes):
+            # Preserved for backward compatibility: allow bytes inputs.
+            glob_str = glob_str.decode("utf-8")
```

Comment:
should we add a unit test for this? i don't know even if it is interesting, just as long as we support this i guess, although we don't list it as a type we support.

Reply:
added because it's simple enough!